src/guides/v2.3/frontend-dev-guide/css-topics/css-overview.md

```diff
@@ -58,6 +58,20 @@ ExampleCorp wants to change the color of the primary buttons to orange. To achie
    @button-primary__hover__border: 1px solid @color-orange-red2;
    ```
 
+ExampleCorp wants to change the color of the primary buttons to custom color. In the Orange theme directory add the overriding app/design/frontend/ExampleCorp/orange/web/css/source/_theme.less file with the following code:
```

Comment:
```suggestion
ExampleCorp wants to change the color of the primary buttons to a custom color. In the Orange theme directory add the overriding `app/design/frontend/ExampleCorp/orange/web/css/source/_theme.less` file with the following code:
```

Reply:
HI @BarnyShergold: 
Thanks for your review and suggestion. Requested suggestion updated.
Adobe CLA signed already and the encryption file downloaded.

Reply:
Encryption file?!

Reply:
@dobooth @BarnyShergold : Encryption file?! What I meant, I have downloaded a copy of the Adobe CLA sign file.